### DISCUSSION TITLE
Colour Blind Theme Changes

### DISCUSSION THREAD

**Discussion by BraddlesUnravels** [community/community#15475] [Accessibility]
Hi GitHub,
I just wanted to say thank you for the change to the colour blind theme being that it now takes in both red/green & blue/yellow now. Not enough companies out there are taking these things into consideration, and I can tell you from personal experience being a mixture of both  red/green & blue yellow colourblind that it makes a big difference.
So thank you for proactivley finding new ways to make your product more inclusive. I wish more companies would be like this.

**Comment by queenofcorgis** [comment]
You're welcome! Thank you for taking the time to send us this note. We're always looking to improve - is there anywhere you think we can enhance our themes?

**Reply by BraddlesUnravels** [reply]
Not off the top of y head right now, but if something comes to me i know where to find this thread :)
